sources/se/municipality_of_kristinehamn.json

```diff
@@ -10,12 +10,12 @@
         "number": {
             "function": "regexp",
             "field": "Name",
-            "pattern": "([0-9]+([A-Z])?)$"
+            "pattern": "^(?:.*?)(?: ([0-9]+[A-Z]?))?$"
```

Comment:
Oh, maybe the captured group shouldn't be optional so it won't return None: "^(?:.*?)(?: ([0-9]+[A-Z]?))$"